base/sparse/linalg.jl

```diff
@@ -941,3 +941,17 @@ end
 chol(A::SparseMatrixCSC) = error("Use cholfact() instead of chol() for sparse matrices.")
 lu(A::SparseMatrixCSC) = error("Use lufact() instead of lu() for sparse matrices.")
 eig(A::SparseMatrixCSC) = error("Use eigs() instead of eig() for sparse matrices.")
+
+function Base.cov(X::SparseMatrixCSC, vardim::Int=1; corrected::Bool=true)
+    a, b = size(X)
+    n, p = vardim == 1 ? (a, b) : (b, a)
+    out = Matrix(Base.unscaled_covzm(X,vardim)) # part1
+    sums = sum(X, vardim)
+    means = mean(X, vardim)
+    @inbounds for j in 1:p, i in 1:p
+        part2 = sums[i]*means[j]
+        part3 = means[i] * means[j] * n
```

Comment:
Notice that `part2 == part3` in exact arithmetic so we can delete either `part2` or `part3`. I think `means[j]` should be `conj`ed here and the `conj` below shouldn't be there.